plugins/inputs/aerospike/aerospike.go

```diff
@@ -121,6 +121,8 @@ func (a *Aerospike) gatherServer(hostport string, acc telegraf.Accumulator) erro
 func parseValue(v string) interface{} {
 	if parsed, err := strconv.ParseInt(v, 10, 64); err == nil {
 		return parsed
+	} else if parsed, err := strconv.ParseUint(v, 10, 64); err == nil {
+		return -int64(parsed)
```

Comment:
oh, now I understand, but this is wrong. The number that you are getting is an overflowed integer, it won't look anything like the actual uint